sections/featured-blog.liquid

```diff
@@ -62,12 +62,17 @@
       {%- endif -%}
     {%- else -%}
       <div class="placeholder">
-        <h2>
-          {{ 'sections.featured_blog.onboarding_title' | t }}
-        </h2>
-        <p class="rte-width">
-          {{ 'sections.featured_blog.onboarding_content' | t }}
-        </p>
+        <div class="placeholder-image">
+          {{ 'image' | placeholder_svg_tag: 'placeholder-svg' }}
```

Comment:
I think you could add the `placeholder` class in here actually. I feel like that's more what it was meant for 🤔  And then the parent can use a new class maybe like `blog-placeholder`. That would also reduce some of the selectors in the CSS (`.blog .placeholder .placeholder-content` vs `blog-placeholder__content`) and keep it specific to this section which it is right now. 

What do you think ?

Reply:
hmm, yeah I see what you mean. To me `.placeholder` doesn't really seem intuitive as representing just the image in the case of placeholder image _and_ text like we have here, but in context, what you suggest is the best bet. Updated.